lib/vtls/openssl.c

```diff
@@ -3317,6 +3317,11 @@ static CURLcode ossl_connect_step2(struct Curl_easy *data,
              error_buffer */
           strcpy(error_buffer, "SSL certificate verification failed");
       }
+      else if((lib == ERR_LIB_SSL) && 
+              (reason == SSL_R_TLSV13_ALERT_CERTIFICATE_REQUIRED)) {
```

Comment:
This sounds TLS 1.3 specific, isn't it?

Reply:
Yes, that is true.

Reply:
So you don't think it's a problem that if you happen to negotiate another TLS version you won't get this error code for the otherwise very similar situation?

Reply:
I would like to add the error code to other TLS versions, but don't know how yet. So, wanted to add concepts first TLS 1.3 and above then slowly add for other versions.